.gitignore

```diff
@@ -1,4 +1,4 @@
 node_modules/
 .vscode/
 *.log
-packages/android-app/assets/js/index.android.bundle.map
+android/assets/js/index.android.bundle.map
```

Comment:
shouldn't this be done for iOS too?

Reply:
Turns out these are ignored inside android and ios folders anyway, removing this one